### ISSUE TITLE
remove lambert albedo from call to Surface

### ISSUE THREAD

**Issue by mjwolff** [kconnour/pyRT_DISORT#11]
### Feature request

it doesn't make sense to have the lambert albedo as a call to Surface since the actual Lambert invocation would be after that:  

    sfc = Surface(0.1, CP.n_streams, CP.n_polar, CP.n_azimuth, OB.user_angles,
                OB.only_fluxes)
    sfc.make_lambertian()

i suggest make the Lambert Albedo a parameter in the make_lambertian method (it can be created with a default value if needed for passing to DISORT, i.e., 

 sfc.make_lambertian(0.1)

**Comment by kconnour** [comment]
I agree but still have a question: does DISORT completely ignore ALBEDO if not using a Lambertian surface? I believe the answer is yes and that it gets all its info from RHOU and RHOQ, but I'm not completely sure. I originally added this as an input so that the user could directly set ALBEDO....  but this may not be relevant. I could just set the albedo to 0 if non-Lambertian surface is used. 

FYI, my original idea to have an abstract surface class is still a good idea (I think) but the methods should really turn into functions... it'd make things cleaner.
